src/resources/breaking-changes.md

```diff
@@ -0,0 +1,828 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists the breaking changes in Dart, organized by release and area.
+Complete release notes are available in the [Dart SDK change log][change log].
+
+According to our [breaking change policy], each listed change contains a link to more
+information on the change, such as migration guidance, background, reasoning,
+caveats, and more.
+
+{{site.alert.note}}
+Changes prior to Dart 3.1.0 and our updated breaking change policy 
+might not have links to additional information. More details on some of those
+changes might be available in their corresponding [change log] entry.
+{{site.alert.end}}
+
+**Dart has two types of breaking changes**:
+* **Standard**: Causes breaks as soon as you [upgrade your sdk version][sdk]. 
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: Causes breaks only when you [upgrade the
+  language version of your library][pubspec] in its `pubspec.yaml`.
+  *These are marked "**Language versioned**"*.
+
+If you have questions or concerns about any of these breaking changes, please 
+[file an issue][]. To be notified about future breaking changes,
+join the [Dart announce][] group.
+
+[breaking change policy]: https://github.com/dart-lang/sdk/blob/master/docs/process/breaking-changes.md
+[change log]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md
+[sdk]: /get-dart
+[pubspec]: /tools/pub/pubspec#sdk-constraints
+[file an issue]: https://github.com/dart-lang/sdk/issues/new
+[Dart announce]: https://groups.google.com/a/dartlang.org/g/announce
+
+## Not yet released to stable
+
+### Libraries
+{: .no_toc}
+
+#### `dart:convert`
+
+* [Changed return types of `utf8.encode()` and `Utf8Codec.encode()`][52801]
+ from `List<int>` to `Uint8List`.
+
+#### `dart:async`
+
+* [Added `interface` modifier to purely abstract classes:][52334]
+ `MultiStreamController`, `StreamConsumer`, `StreamIterator` and `StreamTransformer`.
+
+#### `dart:io`
+
+* [Added `sameSite` to the `Cookie` class, and added the class `SameSite`][51486].
+* [`FileSystemEvent` is `sealed`][52027]. This means `FileSystemEvent` cannot be 
+  extended or implemented.
+
+#### `package:js`
+
+* `external` `@staticInterop` members and `external` extension members can no
+  longer be used as tear-offs. Declare a closure or a non-`external` method that
+  calls these members, and use that instead.
+* `external` `@staticInterop` members and `external` extension members will
+  generate slightly different JS code for methods that have optional parameters.
+
+[52801]: https://github.com/dart-lang/sdk/issues/52801
+[52334]: https://github.com/dart-lang/sdk/issues/52334
+[51486]: https://github.com/dart-lang/sdk/issues/51486
+[52027]: https://github.com/dart-lang/sdk/issues/52027
+
+{% comment %}
+Create new section from these headers for each release.
+Copy entries from "Not yet released to stable". 
```

Comment:
I'm a bit confused on this instruction. Is it talking about moving changes from "Not yet released to stable" to a specific release when it becomes stable? If not, you can ignore the following.

If it is though, we probably should be more specific here as after the beta cut there is a period where a stable version isn't released but breaking changes that are landed aren't part of that coming stable release. Also some changes may have been reverted in that timeline. Essentially there may need to be a bit of a manual verification process.

Reply:
I think I'll remove the instruction to copy from "Not yet released" to the new section (line 73) from this doc. Since there needs to be instructions in the breaking change policy about how to handle all of this anyway, we can elaborate there. 

I'll leave the heading layout for ease of copying in the future.

Lmk if that doesn't address your concern!